website/docs/secrets/manage-secrets-gui.mdx

```diff
@@ -3,18 +3,132 @@ title: Manage Secrets GUI
 hide_title: true
 ---
 
-# Manage Secrets GUI
-// Overview on what the management UI provides for both ESO and sops, 
-// NB. this will change a bit once we get SOPS in the UI, but the content will be the same
-
-## List External Secrets (ESO)
-// Overview + Screenshot
-## External Secret Details (ESO)
-// Overview + Screenshot
-#### Understanding Events
-// Summary about the events to be expected.
-## Create External Secret CR (ESO)
-// Overview + Guide (to include scenarios like namespace/cluster scoped secret stores ) + Screenshot
-## Sync External Secrets
-// Overview + Screenshot
 
+# Manage Secrets UI
+
+At Weave GitOps Enterprise (WGE), we support two approaches for creating and managing secrets: ExternalSecrets Operator (ESO) and SOPS. In this guide, we will provide an overview of both approaches and explain how to use the UI to create and manage secrets.
+
+Clicking on the Secrets in the left hand menu will bring your to the secrets page where you can create external secrets, sops secrets, and view the external secrets list.
+
+![WGE left menu](./imgs/WGE-left-menu.png)
```

Comment:
this image is too narrow and too long, it makes the page look really awkward. I dont feel that it's needed, you're already describing where the user can find it. But if you want to keep it, then at least crop most of it, may be just keep the `Platform` section.